discovery-provider/plugins/notifications/package.json

```diff
@@ -10,7 +10,7 @@
     "sql-ts": "ts-node scripts/sql-ts.ts",
     "start": "pm2-runtime build/src/main.js --restart-delay=3000",
     "test:watch": "jest --watchAll",
-    "test": "jest --runInBand --forceExit",
+    "test": "tsc && jest --runInBand --forceExit",
```

Comment:
whats tsc?

Reply:
typescript compiler

Reply:
yea just to make sure latest changes are built before running tests